### PR TITLE
[raylib/zh-cn] Completed translation

### PR DESCRIPTION
**Pull request by lzw-723**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: zh-cn/raylib-cn.html.markdown [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+lang: zh-cn
+filename: learnraylib-cn.c
+contributors:
+    - ["Nikolas Wipper", "https://notnik.cc"]
+translators:
+    - ["lzw-723", "https://github.com/lzw-723"]
+---
+
+**raylib** 是一个跨平台、易用的图形库，围绕OpenGL 1.1、2.1、3.3和OpenGL ES 2.0构建。
+尽管它是用C语言编写的，却有超过50种不同语言的绑定。本教程将使用C语言。
```
Comment (geoffliu) [inline zh-cn/raylib-cn.html.markdown:13]: 虽然它是？

File: zh-cn/raylib-cn.html.markdown [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+```c
+#include <raylib.h>
+
+int main(void)
+{
+    const int screenWidth = 800;
+    const int screenHeight = 450;
+
+    // 在初始化raylib之前，可以设置配置标志
```
Comment (geoffliu) [inline zh-cn/raylib-cn.html.markdown:24]: 标志位

**Review by geoffliu — APPROVED** [review]
两个小的提议，但是感觉都不很强，尤其是第一个。 @lzw-723 你看要不要改

**Comment by lzw-723** [comment]
> 两个小的提议，但是感觉都不很强，尤其是第一个。 @lzw-723 你看要不要改

感谢指正:)

**Comment by imba-tjd** [comment]
LGTM

**Pull request merged**